include/proxsuite/linalg/veg/memory/alloc.hpp

```diff
@@ -52,6 +52,81 @@ aligned_alloc(std::size_t alignment, std::size_t size)
 } // namespace alignment
 #endif
 
+// Support aligned_alloc for c++14, code from boost/align.
+namespace alignment {
```

Comment:
Could you also add an ifdef for C++ lower than C++17?

Reply:
I did so in the proxsuite/fwd.hpp